src/app/(nextjs_migration)/(guest)/breach-details/[breachName]/page.tsx

```diff
@@ -0,0 +1,368 @@
+              }),
+            }}
+          />
+        </div>
+      </div>
+
+      {/* Sign Up Banner */}
+      <div className="breach-detail-sign-up">
+        <Image alt="" src={BreachDetailScanImage} />
+        <div className="breach-detail-sign-up-content">
+          <h2>{l10n.getString("find-out-if-2")}</h2>
+          <span>{l10n.getString("find-out-if-description")}</span>
+        </div>
+        <div className="breach-detail-sign-up-cta-wrapper">
+          <a
+            href="/user/breaches"
+            data-cta-id="breaches-detail"
+            className="button primary"
+          >
+            {l10n.getString("breach-detail-cta-signup")}
+          </a>
+        </div>
+      </div>
+
+      {/* What to do tips */}
+      <div id="what-to-do-next">
+        <div className="breach-detail-recommendation-lead">
+          <h2>
+            {breach.DataClasses.includes("passwords")
+              ? l10n.getString("rec-section-headline")
+              : l10n.getString("rec-section-headline-no-pw")}
+          </h2>
+          <p>
+            {breach.DataClasses.includes("passwords")
+              ? l10n.getString("rec-section-subhead")
+              : l10n.getString("rec-section-subhead-no-pw")}
+          </p>
+        </div>
+        <dl className="breach-detail-recommendation-list">
+          {makeRecommendationCards(breach)}
+        </dl>
+      </div>
+
+      {/* What is this breach? / Why did it take you so long to report it */}
+      <div className="breach-detail-info">
+        <div id="what-is-this-breach">
+          {makeBreachDetail(getBreachCategory(breach))}
+        </div>
+        {compareBreachDates(breach) ? (
+          <div id="delayed-reporting">
+            <h2>{l10n.getString("delayed-reporting-headline")}</h2>
+            {l10n.getString("delayed-reporting-copy")}
+          </div>
+        ) : null}
+      </div>
+    </div>
+  );
+}
+
+function getBreachDetail(categoryId: Breach) {
+  const l10n = getL10n();
+
+  if (categoryId === "data-aggregator-breach") {
+    return {
+      subhead: l10n.getString("what-is-data-agg"),
+      body: l10n.getString("what-is-data-agg-blurb"),
+    };
+  } else if (categoryId === "sensitive-breach") {
+    return {
+      subhead: l10n.getString("sensitive-sites"),
+      body: l10n.getString("sensitive-sites-copy"),
+    };
+  } else {
+    return {
+      subhead: l10n.getString("what-is-a-website-breach"),
+      body: l10n.getString("website-breach-blurb"),
+    };
+  }
+}
+
+function makeBreachDetail(breach: ReturnType<typeof getBreachCategory>) {
+  const breachDetail = getBreachDetail(breach);
+  return (
+    <>
+      <h2>{breachDetail.subhead}</h2>
+      <div>{breachDetail.body}</div>
+    </>
+  );
+}
+
+function getBreachCategory(breach: Breach) {
+  if (
+    [
+      "Exactis",
+      "Apollo",
+      "YouveBeenScraped",
+      "ElasticsearchSalesLeads",
+      "Estonia",
+      "MasterDeeds",
+      "PDL",
```

Comment:
nit/suggestion: Move this to a variable for legibility. Right now, it's a loose array of breach names where the context is only inferred by the if statement logic.

```js
const dataAggregatorBreach = ["Exactis", "Apollo", "YouveBeenScraped", "ElasticsearchSalesLeads", "Estonia", "MasterDeeds", "PDL"]
```

Reply:
Good suggestion: 1b3818cde